convokit/model/corpus.py

```diff
@@ -33,124 +41,227 @@ class Corpus:
         Type-checking ensures that the ConvoKitIndex is initialized correctly. However, it may be unnecessary if the
         index.json is already accurate and disabling it will allow for a faster corpus load. This parameter is set to
         True by default, i.e. type-checking is not carried out.
+    :param storage_type: either 'mem' or 'db'
+    :param storage: optional convokit.storage.StorageManager object to load the Corpus from
+    :param in_place: when connecting to a Corpus with data already in db storage, whether to load a copy of the Corpus 
+        or modify the existing Corpus in place.
+        True by default, i.e. Corpus is not initialized as a seperate copy.
+    :param from_corpus: if specified, the corpus to make a copy from.
+    :param version: (DB only): What version of the specified corpus to connect to
```

Comment:
for clarity, maybe: "if `storage_type` is 'db', then this specifies the version of the corpus to connect to"